src/vcpkg/commands.dependinfo.cpp

```diff
@@ -320,10 +315,11 @@ namespace vcpkg::Commands::DependInfo
             provider, var_provider, specs, status_db, {host_triplet, UnsupportedPortAction::Warn});
         for (const auto& warning : action_plan.warnings)
         {
-            print2(Color::warning, warning, '\n');
+            msg::write_unlocalized_text_to_stdout(Color::warning, warning + '\n');
         }
-        Checks::check_exit(
-            VCPKG_LINE_INFO, action_plan.remove_actions.empty(), "Only install actions should exist in the plan");
+
+        Checks::msg_check_exit(VCPKG_LINE_INFO, action_plan.remove_actions.empty(), msgInvalidActionsInstall);
```

Comment:
```suggestion
        if (!action_plan.remove_actions.empty()) {
            Debug::println("Only install actions should exist in the plan");
            Checks::unreachable(VCPKG_LINE_INFO);
        }
```

and remove InvalidActionsInstall please.